src/types/stepper.ts

```diff
@@ -1,6 +1,10 @@
-export type StepperState = '' | 'active' | 'default' | 'pending' | 'completed' | 'error'
+import { AnyElementOf } from '@/types/utils'
+
+export const StepperStateArray = ['active', 'default', 'pending', 'completed', 'error'] as const
```

Comment:
If we're following the pattern of the type array containing the empty string as an element, then we could have
`export const StepperStateArray = ['', 'active', 'default', 'pending', 'completed', 'error'] as const`
and `state?: StepperState`

Reply:
Again I don't think we should support `''`, that isn't a valid state

Reply:
removed